build-tools/packages/build-tools/src/bumpVersion/utils.ts

```diff
@@ -57,3 +47,38 @@ export function prereleaseSatisfies(packageVersion: string, range: string) {
 	// Pretend that the current package is latest prerelease (zzz) and see if the version still satisfies.
 	return semver.satisfies(`${packageVersion}-zzz`, range);
 }
+
+/**
+ * Represents the different types of release groups supported by the build tools. Each of these groups should be defined
+ * in the fluid-build section of the root package.json.
+ */
+export enum MonoRepoKind {
+	Client = "client",
+	Server = "server",
+	Azure = "azure",
+	BuildTools = "build-tools",
+	GitRest = "gitrest",
+	Historian = "historian",
+}
+
+/**
+ * A type guard used to determine if a string is a MonoRepoKind.
+ */
+export function isMonoRepoKind(str: string | undefined): str is MonoRepoKind {
+	if (str === undefined) {
+		return false;
+	}
+
+	const list = Object.values<string>(MonoRepoKind);
+	const isMonoRepoValue = list.includes(str);
+	return isMonoRepoValue;
+}
+
+/**
+ * An iterator that returns only the Enum values of MonoRepoKind.
+ */
+export function* supportedMonoRepoValues(): IterableIterator<MonoRepoKind> {
```

Comment:
Once this is completely gone, how do you enumerate the release groups?

Reply:
It will purely be based on release group definitions.